management/src/main/java/org/kaazing/gateway/management/monitoring/entity/impl/AgronaMonitoringEntityFactory.java

```diff
@@ -26,8 +26,8 @@
 import java.util.List;
 import java.util.concurrent.CopyOnWriteArrayList;
 
-import org.kaazing.gateway.management.monitoring.entity.LongMonitoringCounter;
-import org.kaazing.gateway.management.monitoring.entity.factory.MonitoringEntityFactory;
+import org.kaazing.gateway.service.LongMonitoringCounter;
+import org.kaazing.gateway.service.MonitoringEntityFactory;
```

Comment:
Should we have such monitoring entities placed inside gateway.service? Additional entities may need to be made available here from gateway.management, such as DefaultLongMonitoringCounterStub (in order not to have to duplicate the logic for new LongMonitoringCounterStub when creating a COUNTER_STUB in DefaultServiceContext)